config/application.rb

```diff
@@ -14,6 +14,8 @@
 # require "action_cable/engine"
 # require "rails/test_unit/railtie"
 
+require "sprockets/railtie"
```

Comment:
[question] what does this end up doing? I think it should have automatically happened by the `gem "sprockets-rails"` line in `Gemfile`

Reply:
I was not seeing some of the asset pipeline stuff actually kick in (specifically, [rewriting image urls referenced in CSS](https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/rails/asset_url_processor.rb)) until I did this. The sprockets-rails docs say to either add this `require` statement or do `gem 'sprockets-rails', :require => 'sprockets/railtie'` in `Gemfile`. I'm not sure which is "better"?

Reply:
Ah, ok. This is good